### ISSUE TITLE
Mouse (and keyboard) freeze after pressing hotkey.

### ISSUE THREAD

**Issue by illidank** [Barragek0/ClickIt#1]
Sorry for the blur image and probably bad English, but it freeze mouse and keyboard when i press F1, so no screenshot. To be exactly first time I press F1, it does not respond, 2nd time it pick 1 item up, then stop. And after few press (pick one item at a time) it actually automatically picking up all the item, but then freeze both mouse and keyboard in the middle of the work. 
![z3981019139275_9e2070782e423834544876a2a59d3a8f](https://user-images.githubusercontent.com/7278592/209170490-269bc04f-816a-486c-9764-fa54db16a35a.jpg)

**Comment by Barragek0** [comment]
Please use windowed fullscreen for now, pretty sure this is related to windowed mode.

**Comment by illidank** [comment]
> Please use windowed fullscreen for now, pretty sure this is related to windowed mode.

i turn off auto click alta, window is the only way i could play this game T_T Only need to see the weight for fast reading, and i use apikit for picking up

**Comment by Kiritocs** [comment]
> Please use windowed fullscreen for now, pretty sure this is related to windowed mode.

I have the same problem since version 3.20.1. I am using windowed Fullscreen.
[https://youtu.be/pki2H0O9_Bo](url)

**Comment by Barragek0** [comment]
> > Please use windowed fullscreen for now, pretty sure this is related to windowed mode.
> 
> I have the same problem since version 3.20.1. I am using windowed Fullscreen. [https://youtu.be/pki2H0O9_Bo](url)

Could you explain what happens exactly? Does it just keep the mouse and keyboard frozen after it loots that chest?

**Comment by Kiritocs** [comment]
> > > Please use windowed fullscreen for now, pretty sure this is related to windowed mode.
> > 
> > 
> > I have the same problem since version 3.20.1. I am using windowed Fullscreen. [https://youtu.be/pki2H0O9_Bo](url)
> 
> Could you explain what happens exactly? Does it just keep the mouse and keyboard frozen after it loots that chest? Also, are you holding your key to pickup for that entire clip, or are you letting go of it mid-way through the video?

This happens when using any of the clickit functions (pick up items, open chests, click shrine/harvest/essence). Here is another clip that demonstrates this:
https://youtu.be/fdJeLyeRN5U

In this clip, the mouse and keyboard simply freeze as soon as there is nothing left to click on. Sometimes mouse and keyboard also freeze before anything is clicked. 
I kept pressing the hotkey until it froze.

I also deleted all clickit settings before the video and used the default settings.

**Comment by Barragek0** [comment]
So it isn't unfreezing the mouse/keyboard correctly when you let go of the key? It just keeps them frozen even when letting go of it?

**Comment by Kiritocs** [comment]
> So it isn't unfreezing the mouse/keyboard correctly when you let go of the key? It just keeps them frozen even when letting go of it?
> 
Correct, it does not unfreeze when I release the key. It takes about a minute or two until mouse/keyboard is working again.

> Edit: If that is the case, does it happen with normal pickit too? Or only this?
> 
It does not happen with normal PickIt. As I said in a previous post, the mouse/keyboard freeze problem happens since version 3.20.1.
I've been using ClickIt for over 5 Months now :)

> Edit2: Sorry if i'm misunderstanding the issue, but for the plugin to pick up items/interact with chests/other things, they have to be on the screen. It won't automatically move to them, you have to move to them yourself, then when they're on the screen, hold the key. If this is what you're doing, and it's an issue with the plugin, disregard this, I just want to make sure that it's not miscommunication of how the plugin works.

**Comment by Barragek0** [comment]
> > So it isn't unfreezing the mouse/keyboard correctly when you let go of the key? It just keeps them frozen even when letting go of it?
> 
> Correct, it does not unfreeze when I release the key. It takes about a minute or two until mouse/keyboard is working again.
> 
> > Edit: If that is the case, does it happen with normal pickit too? Or only this?
> 
> It does not happen with normal PickIt. As I said in a previous post, the mouse/keyboard freeze problem happens since version 3.20.1. I've been using ClickIt for over 5 Months now. Pretty much since you published it on Github.
> 
> > Edit2: Sorry if i'm misunderstanding the issue, but for the plugin to pick up items/interact with chests/other things, they have to be on the screen. It won't automatically move to them, you have to move to them yourself, then when they're on the screen, hold the key. If this is what you're doing, and it's an issue with the plugin, disregard this, I just want to make sure that it's not miscommunication of how the plugin works.

For now, try disabling block user input in your settings, as it's most likely related to that. Will take a look later today or tomorrow to see if I can figure out what's causing it. It doesn't seem to be happening for most users though, including me, so i'm not sure if i'll be able to.

**Comment by Kiritocs** [comment]
> > > So it isn't unfreezing the mouse/keyboard correctly when you let go of the key? It just keeps them frozen even when letting go of it?
> > 
> > 
> > Correct, it does not unfreeze when I release the key. It takes about a minute or two until mouse/keyboard is working again.
> > > Edit: If that is the case, does it happen with normal pickit too? Or only this?
> > 
> > 
> > It does not happen with normal PickIt. As I said in a previous post, the mouse/keyboard freeze problem happens since version 3.20.1. I've been using ClickIt for over 5 Months now. Pretty much since you published it on Github.
> > > Edit2: Sorry if i'm misunderstanding the issue, but for the plugin to pick up items/interact with chests/other things, they have to be on the screen. It won't automatically move to them, you have to move to them yourself, then when they're on the screen, hold the key. If this is what you're doing, and it's an issue with the plugin, disregard this, I just want to make sure that it's not miscommunication of how the plugin works.
> 
> For now, try disabling block user input in your settings, as it's most likely related to that. Will take a look later today or tomorrow to see if I can figure out what's causing it. It doesn't seem to be happening for most users though, including me, so i'm not sure if i'll be able to.

Disabling "Block User Input" fixed it for now.

**Comment by Barragek0** [comment]
Fixed in v3.20.1.5

**Comment by Barragek0** [comment]
Reopening the issue as it seems to still be happening for some users

**Comment by Barragek0** [comment]
I wasn't able to reproduce the issue at all, so i'm going to close this. If you're experiencing this problem, disable 'block user input' as a workaround, or if you have more information on what could be causing this, please reopen this issue with more information.
